src/localisation/string_ids.h

```diff
@@ -2085,7 +2085,7 @@ enum {
 
 	STR_EDIT_TITLE_SEQUENCES_BUTTON = 5436,
 
-	STR_MININISE_FULL_SCREEN_ON_FOCUS_LOSS = 5440,
+	STR_MININISE_FULLSCREEN_ON_FOCUS_LOSS = 5440,
```

Comment:
Guess that means ```settings.ini``` should be tweaked at some point. Probably a job for a separate PR